### PR TITLE
Update refs source url

### PR DESCRIPTION
**Pull request by adamaltman**

### CODE REVIEW

File: openapi/components/securitySchemes/JWT.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 description: >
   You can create a JSON Web Token (JWT) via our [JWT Session
-  resource](https://user-api-docs.rebilly.com/#operation/PostSigninRequest).
+  resource](https://api.redoc.ly/registry/rebilly/api-reference-users/users/bundle/master/#operation/PostSigninRequest).
```
Comment (ashkarpetin) [inline openapi/components/securitySchemes/JWT.yaml:3]:
```suggestion
  resource](https://api.redoc.ly/registry/rebilly/api-reference-users/users/bundle/master/openapi.yaml#operation/PostSigninRequest).
```

File: openapi/components/securitySchemes/PublishableApiKey.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ description: >
   Only for the Tokens resource.
 
   You can create a Publishable API Key via our [API Keys
-  resource](https://user-api-docs.rebilly.com/#tag/API-Keys), by
+  resource](https://api.redoc.ly/registry/rebilly/api-reference-users/users/bundle/master/#tag/API-Keys), by
```
Comment (ashkarpetin) [inline openapi/components/securitySchemes/PublishableApiKey.yaml:5]:
```suggestion
  resource](https://api.redoc.ly/registry/rebilly/api-reference-users/users/bundle/master/openapi.yaml#tag/API-Keys), by
```

**Comment by RomanHotsiy** [comment]
@VladimirBerdnik This repo is connected to staging version of Redocly so it didn't have access to the API. I just make API public in registry so it can access now.

File: openapi/components/securitySchemes/JWT.yaml (hunk 1 above)
Comment (ashkarpetin) [reply openapi/components/securitySchemes/JWT.yaml]: This was incorrect

File: openapi/components/securitySchemes/PublishableApiKey.yaml (hunk 2 above)
Comment (ashkarpetin) [reply openapi/components/securitySchemes/PublishableApiKey.yaml]: This was incorrect suggestion

File: openapi/components/securitySchemes/PublishableApiKey.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ description: >
   Only for the Tokens resource.
 
   You can create a Publishable API Key via our [API Keys
-  resource](https://user-api-docs.rebilly.com/#tag/API-Keys), by
+  resource](https://api.redoc.ly/registry/rebilly/api-reference-users/users/bundle/master/openapi.yaml#tag/API-Keys), by
```
Comment (ashkarpetin) [inline openapi/components/securitySchemes/PublishableApiKey.yaml:5]:
```suggestion
  resource](https://user-api-docs.rebilly.com/#tag/API-Keys), by
```

File: openapi/components/securitySchemes/JWT.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 description: >
   You can create a JSON Web Token (JWT) via our [JWT Session
-  resource](https://user-api-docs.rebilly.com/#operation/PostSigninRequest).
+  resource](https://api.redoc.ly/registry/rebilly/api-reference-users/users/bundle/master/openapi.yaml#operation/PostSigninRequest).
```
Comment (ashkarpetin) [inline openapi/components/securitySchemes/JWT.yaml:3]:
```suggestion
  resource](https://user-api-docs.rebilly.com/#operation/PostSigninRequest).
```

**Pull request merged**